container_images/test-manager/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export CGO_ENABLED=0 GOOS=linux
+BUILD_DIR=./test_manager
```

Comment:
doesn't need ./, don't even need this variable since we only use it once, on next line

Reply:
remove